addons/mail/static/src/owl/components/composer_text_input/composer_text_input.xml

```diff
@@ -1,9 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
+    <t t-name="owl.UserStatus" owl="1">
+        <span t-att-class="partnerID ? 'o_mention_updatable_im_status' : ''" t-att-data-partner-id="partnerID">
```

Comment:
we probably want a local id instead (which might also imply converting the result of the rpc into local id, but I still need to check that in more details)